src/components/Header/SearchBar.component.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { BiSearch } from 'react-icons/bi';
+import { StyledSearchBar } from '../../styled/Header.styled';
+
+const SearchBar = ({ keyword, setKeyword }) => {
+  const handleChange = (e) => {
+    e.preventDefault();
+    setKeyword(e.target.value);
+  };
+  return (
+    <StyledSearchBar>
+      <BiSearch />
+      <input
+        type="text"
+        placeholder={keyword}
+        id="header-search"
+        onChange={(e) => handleChange(e)}
```

Comment:
I would recommend passing the reference of the function, technically speaking will do the same but in my opinion, it looks cleaner.

```suggestion
        onChange={handleChange}
```